src/fides/api/models/messaging_template.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Dict
+
+from pydantic import BaseModel
+from sqlalchemy import Column, String
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.ext.mutable import MutableDict
+
+from fides.api.db.base_class import Base
+from fides.api.schemas.messaging.messaging import MessagingActionType
+
+# Provides default values for initializing the database or replacing deleted values for messaging templates.
+# Note: There are additional MessagingActionTypes that are internally used but are not exposed for user customization.
+DEFAULT_MESSAGING_TEMPLATES: Dict[str, Any] = {
```

Comment:
super nit:

I'm also not sure what the right location is for this either. This file is probably fine but it's not a model.

Reply:
Keeping here for now